services/web-app/components/gif-player/gif-player.module.scss

```diff
@@ -0,0 +1,54 @@
+@use '@carbon/react/scss/theme' as theme;
+
+.container {
+  position: relative;
+}
+
+.controls {
+  position: absolute;
+  bottom: 1rem;
+  left: 1rem;
+  width: 1.5rem;
+  height: 1.5rem;
+  padding: 0;
+  border: none;
+  background: transparent;
+  outline: none;
+}
+
+.controls svg {
+  fill: #ffffff;
```

Comment:
Lets replace this with a token, I believe `$icon-inverse` would be appropriate here.

Reply:
Done!